### ISSUE TITLE
[CSSWG] PR request for Selectors 3

### ISSUE THREAD

**Issue by svgeesus** [w3c/transitions#83]
# Document title, URLs, estimated publication date
Selectors Level 3
https://www.w3.org/TR/selectors-3/ (as CR)
Tues/Thurs after transition mtg

# Abstract
https://www.w3.org/TR/selectors-3/#abstract

# Status
https://www.w3.org/TR/selectors-3/#abstract

# Link to group's decision to request transition
(expected on 22 August CSS WG call)

# Changes
none

# Requirements satisfied
same as before, this is the Selectors 3 Rec plus errata

# Dependencies met (or not)
no change

# Wide Review
CR issued 30 January 2018, no comments.

# Issues addressed
No issues

# Formal Objections
none

# Implementation
All browsers, both in CSS and also in the DOM

# Patent disclosures
none, see https://www.w3.org/2004/01/pp-impl/32061/status
no open exclusion opportunity

**Comment by svgeesus** [comment]
CSS WG decision just now

> RESOLVED: publish Selectors 3 as PR
https://www.w3.org/2018/08/22-css-irc#T16-32-37

**Comment by swickr** [comment]
> Implementation
> All browsers, both in CSS and also in the DOM

The [test results](http://test.csswg.org/harness/details/selectors-3_dev/selector-structural-pseudo-root/) say that Edge is failing.   Any further info?

**Comment by fantasai** [comment]
@swickr I don't think we have all implementations passing yet, just most of them, but sent a message asking MSFT if they have any test builds that pass yet.

**Comment by fantasai** [comment]
No passes from MSFT yet.

**Comment by swickr** [comment]
Transition approved.
